collections/src/test/scala/strawman/collection/test/Test.scala

```diff
@@ -465,7 +465,7 @@ class StrawmanTest {
     var lazeCountL = 0
     def lazeL(i: Int) = {lazeCountL += 1; i}
     def lazeS(i: Int) = {lazeCountS += 1; i}
-    val xs20 = lazeS(1) #:: lazeS(2) #:: lazeS(3) #:: old.Stream.empty
+    val xs20 = lazeS(1) #:: lazeS(2) #:: lazeS(3) #:: Stream.empty
     val xs21 = lazeL(1) #:: lazeL(2) #:: lazeL(3) #:: LazyList.empty
     assert(lazeCountS==lazeCountL)
```

Comment:
Since `Stream` and `LazyList` don’t have the same laziness level this assertion will never be fulfilled.